client/package.json

```diff
@@ -18,6 +18,7 @@
 		"@automattic/components": "^1.0.0-alpha.1",
 		"@automattic/composite-checkout": "^1.0.0",
 		"@automattic/data-stores": "^1.0.0-alpha.1",
+		"@automattic/domain-picker": "1.0.0-alpha.0",
```

Comment:
How will work-flow with these versions looks like? Would every PR touching the code need to bump the version?

Can we use `*` here instead?

Reply:
`^1.0.0-alpha.0` or `*` shouldn't make a difference in this case.

Because this plugin in the same monorepo, `yarn.lock` ignores it. So this version will be resolved each time we do `yarn install`. And as long as the version in `packages/domain-picker/package.json` satisfies this range, it should work without any change.

Reply:
Excellent 👍 and since FSE plugin (the only other consumer of this package) is in the monorepo too, we can remove `version` from `package.json` altogether. ([It's not mandatory](https://docs.npmjs.com/files/package.json#version))

Reply:
Now uses `^`.